sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,5 @@
+select count("cities") as "numberOfCities", "countries"."name" as "country"
```

Comment:
`count(*)` is more traditional, but it doesn't matter. `count()` ignores its argument!